src/nimvscode/nimBuild.nim

```diff
@@ -112,64 +120,56 @@ proc nimExec(
     )
 
 proc parseErrors(lines:seq[cstring]):seq[CheckResult] =
-    var ret:seq[CheckResult] = @[]
-    var messageText = ""
-    var lastFile:cstring = ""
-    var lastColumn:cstring = ""
-    var lastLine:cstring = ""
+    var
+        messageText = ""
+        stacktrace:seq[CheckStacktrace]
 
     # Progress indicator from nim CLI is just dots
     var dots = newRegExp(r"^\.+$")
-    for line in lines.mapIt(it.strip()).filterIt(
-        not(it.startsWith("Hint:") or it == "" or dots.test(it))
-    ):
-        var match = newRegExp(r"^([^(]*)?\((\d+)(,\s(\d+))?\)( (\w+):)? (.*)")
+    for line in lines:
+        let line = line.strip()
+
+        if line.startsWith("Hint:") or line == "" or dots.test(line):
+            continue
+
+        console.log(line)
```

Comment:
Don't mind leaving the console log line here, but can you make the first parameter a some descriptive constant string `"nimBuild - parseErrors:"` so it can be filtered for or out in development?